dotcom-rendering/src/components/FrontSection.tsx

```diff
@@ -513,6 +513,15 @@ export const FrontSection = ({
 		!!pageId &&
 		!!ajaxUrl;
 
+	const frontHasEditorialOrDesignBadge =
+		!isOnPaidContentFront && containerBranding === undefined;
+
+	// Only show the badge with a "Paid for by" label on the FIRST card of a paid front, aka as Labs front.
+	const isTheFirstContainerOnAPaidFront = isOnPaidContentFront && index === 0;
+
+	const hasFrontBranding =
+		!isOnPaidContentFront && !!frontBranding && index === 0;
```

Comment:
Creating these booleans to improve readability as suggested by @arelra in https://github.com/guardian/dotcom-rendering/pull/8764#discussion_r1319732633